include/mcl_3dl/parameters.h

```diff
@@ -200,6 +202,10 @@ class Parameters
   std::shared_ptr<PointCloudSamplerWithNormalParameters> random_sampler_with_normal_params_;
   std::shared_ptr<LidarMeasurementModelLikelihoodParameters> lidar_measurement_likelihood_params_;
   std::shared_ptr<LidarMeasurementModelBeamParameters> lidar_measurement_beam_params_;
+
+private:
+  std::unique_ptr<dynamic_reconfigure::Server<MCL3DLParamsConfig>> parameter_server_;
+  void cbParameter(const MCL3DLParamsConfig& config, const uint32_t /* level */);
```

Comment:
- note: these the ROS parameter server thing will be moved to the ROS node class for https://github.com/at-wat/mcl_3dl/issues/433
  As the parameter values are directly loaded from ROS paramters for now, having them here would be ok.